alcs-api/src/decision/application-decision/application-decision.service.ts

```diff
@@ -154,8 +154,10 @@ export class ApplicationDecisionService {
     await this.validateDecisionChanges(updateDto);
 
     if (
-      existingDecision.resolutionNumber !== updateDto.resolutionNumber ||
-      existingDecision.resolutionYear !== updateDto.resolutionYear
+      updateDto.resolutionNumber &&
+      updateDto.resolutionYear &&
+      (existingDecision.resolutionNumber !== updateDto.resolutionNumber ||
+        existingDecision.resolutionYear !== updateDto.resolutionYear)
```

Comment:
Does this additional check do anything? If the updateDto.resolutionNumber or resolutionYear was undefined it would not equal the numbers in resolutionNumber/resolutionYear.

Reply:
yep, the validate function does not care about the undefined values. Since the typeorm ignores the undefined value on update we do not need to run validation against it